File: LeetCode-Programs/Minimum Cost to Move Chips to The Same Position.cpp
class Solution {
public:
    int minCostToMoveChips(vector<int>& position) {
        int e_count=0;
        int o_count=0;
        for(int i=0;i<position.size();i++)
        {
            if(position[i]%2==0)
                e_count++;
            else
                o_count++;
        }
        return min(e_count, o_count);
        
    }
};

File: LeetCode-Programs/Minimum Cost to Move Chips to The Same Position.py
class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        e_count=0
        o_count=0
        for i in range(len(position)):
            if position[i]%2==0:
                e_count=e_count+1
            else:
                o_count=o_count+1
        return min(e_count, o_count)
      
      
      
      
      
#different way of writing above code
class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        e_count=0
        o_count=0
        for i in position:
            if i%2==0:
                e_count=e_count+1
            else:
                o_count=o_count+1
        return min(e_count, o_count)
